src/interface.js

```diff
@@ -0,0 +1,33 @@
+let array = []
+
+document.addEventListener("DOMContentLoaded", () => {
+  console.log('Page fully loaded and parsed.')
+  fetchFromApi()
+
+  showArticles = () => {
+    for (let i = 0; i < newsArticles.length; i++) {
+      array += `<a href=${newsArticles[i]._webUrl} style="text-decoration: none;">
```

Comment:
took me ages to figure out that this was the best way of adding stuff in a for loop. Nice idea to add the style tag in here too save you time later on